common/static/common/js/discussion/views/discussion_inline_view.js

```diff
@@ -207,7 +207,7 @@
             if (this.showed) {
                 this.$('section.discussion').find('.inline-discussion-thread-container').addClass('is-hidden');
                 this.$('section.discussion').find('.add_post_btn_container').addClass('is-hidden');
-                this.newPostForm.removeClass('is-hidden').find('.js-post-title');
+                this.newPostForm.removeClass('is-hidden').find('.post-type-input:checked').focus();
```

Comment:
This call to `.focus()` was intentionally removed in the inline discussion PR as an a11y issue. It looks to have found its way back when you fixed the merge conflicts.